packages/shared/src/components/post/SharePostContent.tsx

```diff
@@ -69,7 +70,10 @@ function SharePostContent({
                 className="mt-5 btn-secondary w-fit"
                 href={post.sharedPost.permalink}
                 openNewTab={openNewTab}
-                onClick={onReadArticle}
+                title="Go to post"
+                rel="noopener"
+                onClick={(e) => e.stopPropagation()}
```

Comment:
This shouldn't be needed right?
As combinedClicks also adds a onClick handler already.